src/Api/index.js

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+
+
+const instance = axios.create({
+    baseURL: 'http://localhost:3000'
+})
+
+
+instance.interceptors.request.use(function (config) {
+    console.log('Request sent successfully');
+    return config;
+}, function (err) {
+    console.log(err.status)
+    return Promise.reject(err);
+})
+
+instance.interceptors.response.use(function (response) {
+    console.log(response.status);
+    return response;
+}, function (err) {
+    console.log(err.message);
+    return Promise.reject(err);
+})
+
+export const callUserLoginApi = (enteredUsername, enteredPassword) => {
+    return instance.post('/login', {
+        username: enteredUsername,
+        password: enteredPassword,
+    });
+
+}
+
+export const callUserSignupApi = (enteredUsername, enteredPassword, enteredEmail) => {
+    return instance.post('/signup', {
+        username: enteredUsername,
+        password: enteredPassword,
+        email: enteredEmail
+    });
+}
+
+export const callCreateNoteApi = (id, title, description, date,) => {
+    return instance.post('/article', {
+        id: id,
```

Comment:
You can use the shorthand syntax in cases like this `{id, title, description,date}`